packages/design-carousel/src/index.tsx

```diff
@@ -25,10 +25,18 @@ export default function DesignCarousel( { onPick }: Props ) {
 		_locale: locale,
 	} );
 
-	const staticDesigns = allDesigns?.static?.designs.slice( 0, 10 ) || [];
+	// Cherry-picked designs
+	const designSlugs = [ 'tsubaki', 'tazza', 'thriving-artist', 'twentytwentytwo' ];
```

Comment:
It may be premature, but it may be worth adding `zaino` here so we automatically get that theme in this flow once it launches.